tests/prefect/test_tasks_sqlite.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import pandas
+from viadot.tasks.sqlite_tasks import RunSQL, LoadDF
+from collections import defaultdict
+import pandas as pd
+import os
+
+TABLE = "test"
+DB_PATH = ':memory:'
+SQL_PATH = "home/viadot/tests/testfile.sql"
+
+@pytest.fixture(scope="session")
+def load_table():
+    load_table = LoadDF()
+    yield load_table
+
+
+def test_create_table_from_df(load_table):
+    dtypes = {"country": "VARCHAR(100)", "sales": "FLOAT(24)"}
+    data = defaultdict(list)
+    data["country"].append('italy')
+    data["sales"].append(100.0)
+    df_data = pd.DataFrame(data)
```

Comment:
please create `data` as a normal dict of arrays, you don't need to use append, do something like `data = {"sales":[100,200]}`